p5/io/http.py

```diff
@@ -15,6 +22,15 @@ def http_get(url):
 
 #synchronous
 def http_post(url,post_data):
+	"""
+	Performs a synchronous POST request on the given url.
+
+	:param url: url of the API
+	:type url: string
+	:param post_data: Key-value pairs of the request payload.
+	:type post_data: map 
```

Comment:
Maybe this should be a `dict` since it's in Python?

Reply:
Ok I ll make the change. :)